zgw_consumers/contrib/setup_configuration/steps.py

```diff
@@ -0,0 +1,34 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import SelfTestFailed
+
+from zgw_consumers.models import Service
+
+from .models import ServicesConfigurationModel
+
+
+class ServiceConfigurationStep(BaseConfigurationStep[ServicesConfigurationModel]):
+    """
+    Configure Services to connect with external APIs
+    """
+
+    verbose_name = "Configuration to connect with external services"
+    config_model = ServicesConfigurationModel
+    namespace = "zgw_consumers"
```

Comment:
I find it confusing that the keys are uppercase in the yaml but not here, especially because otherwise case-sensitivity matters. Can we make this consistent?

Reply:
This should be the lowercase now